.travis.yml

```diff
@@ -13,56 +13,84 @@ notifications:
 cache:
   directories:
     - $HOME/.composer/cache
```

Comment:
I believe this is for older versions of Composer, see [WPCS#1435](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1435/files) for reference, so maybe:
```diff
-    - $HOME/.composer/cache
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
```

Reply:
I've confirmed that this is correct by running `composer config cache-dir` in the CI environment and getting `/home/travis/.composer/cache`.

Reply:
Right, all 7 CI jobs in this PR use `Composer version 1.5.2`, the latest is 1.8.0.

When I run `$ echo $(composer config cache-dir)` on one of my CI jobs, it's using `Composer version 1.8.0` and the cache directory is `/home/travis/.cache/composer`

Bumping the environemnt from Trusty to Xenial might be worth exploring here in a follow up PR